stripe/src/test/java/com/stripe/android/EphemeralKeyManagerTest.java

```diff
@@ -250,4 +251,48 @@ public void updateKeyIfNecessary_whenReturnsError_setsExistingKeyToNull() {
         verifyNoMoreInteractions(mKeyManagerListener);
         assertNull(keyManager.getEphemeralKey());
     }
+
+    @Test
+    public void triggerCorrectErrorOnInvalidRawKey() {
+
+        mTestEphemeralKeyProvider.setNextRawEphemeralKey("Not_a_JSON");
+        EphemeralKeyManager<CustomerEphemeralKey> keyManager = new EphemeralKeyManager(
+                mTestEphemeralKeyProvider,
+                mKeyManagerListener,
+                TEST_SECONDS_BUFFER,
+                null,
+                CustomerEphemeralKey.class);
+
+        verify(mKeyManagerListener, times(0)).onKeyUpdate(
```

Comment:
you can do `never()` instead of `times(0)`

Reply:
👍